web/hand_tool.js

```diff
@@ -46,12 +43,12 @@ var HandTool = (function HandToolClosure() {
 
     this.eventBus.on('togglehandtool', this.toggle.bind(this));
 
-    Promise.all([localized,
-                 preferences.get('enableHandToolOnLoad')]).then((values) => {
+    let enableOnLoad = preferences.get('enableHandToolOnLoad');
+    Promise.all([localized, enableOnLoad]).then((values) => {
```

Comment:
Note that you can actually use array destructuring here (and I'm not sure that the temporary `enableOnLoad` is necessary for readability); how about the following instead?
```javascript
Promise.all([
  localized,
  preferences.get('enableHandToolOnLoad')
]).then(([, value]) => {
  if (value === true) {
    this.handTool.activate();
  }
}).catch(function(reason) {});
```